components/extractors/ip-addresses/kbc-public-ip.json

```diff
@@ -32,6 +32,18 @@
             "region": "us-east-1",
             "service": "syrup"
         },
+        {
+            "ipPrefix": "52.7.83.136/32",
+            "vendor": "aws",
+            "region": "us-east-1",
+            "service": "job-queue"
```

Comment:
Je to ok takhle?

Reply:
ve `/v2/storage` to máme pojmenovaný jako `queue`

Reply:
ok upravim na `queue`